src/res/flame.html

```diff
@@ -258,6 +281,11 @@ <h1>/*title:*/</h1>
 		search(true);
 	}
 
+	document.getElementById('legend').onclick = function() {
```

Comment:
What if a legend appears automatically when hovering over? Won't it be a better UX?

Reply:
I imagine accidental appearances would not be helpful. Other buttons in the same area work with a click, I think this is consistent.